### DISCUSSION TITLE
Revert removal of volume number before book titles

### DISCUSSION THREAD

**Discussion by Jokey665** [Kareadita/Kavita#2901] [Ideas]
### Idea Description

Previously in book libraries the titles would have the volume number listed before the title.

Previous
"0.5 The Hobbit"
"1 The Fellowship of the Ring"
etc.

Current
"The Hobbit"
"The Fellowship of the Ring"
etc.

I would like an option to restore the previous behavior.

### Idea Category

User Experience

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by StevieCoops** [comment]
Upvoted this. It's a nice UI feature that would let you know which books were major entries vs the novella. It also allows you to see which volumes are missing faster

**Comment by Nebelherr23** [comment]
Why not make it a toggle? It would allow for people who'd want the chapters gone to have theirs while those who want to keep the numbers to have it that way.

**Comment by StevieCoops** [comment]
Any update on this being added back?
